.github/workflows/cura-installer-windows.yml

```diff
@@ -108,6 +108,9 @@ jobs:
           conan config install https://github.com/Ultimaker/conan-config.git -a "-b runner/internal"
           conan user -p ${{ secrets.CONAN_PASS }} -r cura-private ${{ secrets.CONAN_USER }}
 
+      - name: Remove Cura Packages (Powershell)
```

Comment:
this seems like a debug cache step, is this still needed

Reply:
right It was a debug step. I'll remove it

Reply:
[635b415](https://github.com/Ultimaker/cura-workflows/pull/9/commits/635b4158d2c789098ddfef6dd11f1a741e0da32a)